validateFileNames.js

```diff
@@ -0,0 +1,94 @@
+const shelljs = require('shelljs');
+
+// Regex that defines our file and directory naming convention
+const camelCasedFileNameRegex = /^[.]?([a-z])+([0-9]|[a-zA-Z]|[.])*$/;
+
+// Files and directories that are exempt from the naming convention
+const ignoredFilesAndDirectories = [
+  'README.md',
+  'Dockerfile',
+  'LICENSE.md',
+  'customTypings',
+  'typings',
+];
```

Comment:
I had the same idea. We can probably start by checking for the ignore file in `process.cwd()`, if it does not exist, we fall back to this array.